lib/node_modules/@stdlib/stats/strided/ztest2/docs/repl.txt

```diff
@@ -0,0 +1,144 @@
+
+{{alias}}( NX, NY, alternative, alpha, diff, sigmax, sigmay, x, strideX, y, strideY, out )
```

Comment:
```suggestion
{{alias}}( NX, NY, alt, alpha, diff, sigmax, sigmay, x, sx, y, sy, out )
```

Use shorter parameter names when the signature goes beyond 80 chars. You'll need to update the parameter list below with the updated names.